devdoc/interlocked_requirements.md

```diff
@@ -0,0 +1,298 @@
+# interlocked
+================
+
+## Overview
+
+`interlocked` exports platform independent atomic operations. It is a platform abstraction and it requires a specific implementation for each platform.
+
+The interface is based on the Windows Interlocked API.
```

Comment:
>Windows Interlocked API [](start = 30, length = 23)

if you feel like it (since there is at least one other fix to make), you could even link to the docs in the markdown: [https://docs.microsoft.com/en-us/windows/win32/sync/interlocked-variable-access](https://docs.microsoft.com/en-us/windows/win32/sync/interlocked-variable-access)